pwa-devdocs/src/venia-ui/reference/components/Shimmer/index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Shimmer
+---
+
+# Shimmer
+
+The Shimmer component is a loading indicator that takes the shape of the component being loaded.
+Instead of blocking the entire page like a traditional full-screen loader, Shimmer loaders are component-shape specific to show users previews of what's loading on the page.
+
+These previews improves the perceived performance of the app and prevents CLS (Content Layout Shift).
+The Shimmer component eliminates most of the CLS on a page, which helps improve Google Lighthouse scores.
+
+<!--
+The reference doc content is generated automatically from the source code.
+To update this section, update the doc blocks in the source code
+-->
+
+{% include auto-generated/venia-ui/lib/components/Shimmer/shimmer.md %}
+
+## Accessibility
```

Comment:
I would probably throw this section down lower. It's kind of "extra" information